### PR TITLE
Fixed lint errors in k8s/from_yaml.go file

### PR DESCRIPTION
**Pull request by rbo13**
**What this PR does / why we need it**:
- Fixes all the linting issue from: https://github.com/openebs/maya/blob/master/pkg/k8s/from_yaml.go

**Which issue this PR fixes**
- https://github.com/openebs/openebs/issues/1931

Signed-off-by: Richard Burk Orofeo <chardy.orofeo@yahoo.com.ph>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/k8s/from_yaml.go [hunk 1]
```diff
@@ -14,11 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// instead of pkg/maya/maya.go
+// Package k8s defines
+// all the k8s related
+// codebase
```
Comment (sonasingh46) [inline pkg/k8s/from_yaml.go]: Can we put these in a single line.

File: pkg/k8s/from_yaml.go [hunk 2]
```diff
@@ -35,6 +38,9 @@ type DeploymentYml struct {
 	YmlInBytes []byte
 }
 
+// NewDeploymentYml setups the given
+// context, yml, values and returns a
+// pointer to DeploymentYml, otherwise an error
```
Comment (sonasingh46) [inline pkg/k8s/from_yaml.go]: here also

**Comment by sonasingh46** [comment]
@rbo13 
Thanks for the PR!
Can we put the comments in single line and break only after it goes out of page scope.
Additionally, adding a period at the end of comment line would be great.

File: pkg/k8s/from_yaml.go (hunk 1 above)
Comment (rbo13) [reply pkg/k8s/from_yaml.go]: Will do @sonasingh46 👍 thanks for the feedback.

**Comment by rbo13** [comment]
Hi @sonasingh46, already addressed the following comments above. Please check it :)

File: pkg/k8s/from_yaml.go [hunk 3]
```diff
@@ -14,11 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// instead of pkg/maya/maya.go
+// Package k8s defines all the k8s related codebase.
```
Comment (sonasingh46) [inline pkg/k8s/from_yaml.go]: @princerachit 
Please give some insights here.
I feel this comment can be improved.

Comment (rbo13) [reply pkg/k8s/from_yaml.go]: Okay @sonasingh46 👍

Comment (rbo13) [reply pkg/k8s/from_yaml.go]: Hi @sonasingh46, I've updated the partial improvement of the comment, hopefully @princerachit can give some insights/feedback :) thank you so much!

File: pkg/k8s/from_yaml.go [hunk 4]
```diff
@@ -14,11 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// instead of pkg/maya/maya.go
+// Package k8s provides the necessary codebase for K8s implementation.
+// Setups the different yaml files for deployment
+
```
Comment (sonasingh46) [inline pkg/k8s/from_yaml.go]: @rbo13 
Can you please remove the comment ( // Package k8s ....) and just keep a blank line there.
That would do fine.
Sorry for bugging you.
(e.g. At line no 15 comments end... line no 16 should be a blank line and `package k8s` should be there on line no 17)

Comment (rbo13) [reply pkg/k8s/from_yaml.go]: Okay @sonasingh46 👍 its fine :) will update my PR

**Comment by sonasingh46** [comment]
@rbo13 
Can you please rebase to master ( instead to doing a git pull)
I can see a lot commits propagated to your PR.

**Comment by rbo13** [comment]
Already did `rebase` from master ( which is pointing to openebs/maya.git ).

Currently I have this lists:

screen shot 2018-10-10 at 09 58 35

I'm still seeing lots of commit being propagated. What will I do @sonasingh46?

**Comment by princerachit** [comment]
@rbo13 Can you raise the same Pr again from a different branch(not rbo13:master)?
Please pull the changes into `openebs:master`

**Comment by rbo13** [comment]
Thanks @princerachit 👍 will follow as instructed

**Comment by rbo13** [comment]
Hi @sonasingh46, I've created a new PR as suggested by @princerachit. You can check it here: https://github.com/openebs/maya/pull/642

**Comment by kmova** [comment]
Thanks for raising the new PR @rbo13

**Pull request closed without merging**